inlong-tubemq/tubemq-client-twins/tubemq-client-go/log/config.go

```diff
@@ -39,3 +39,18 @@ var defaultConfig = &OutputConfig{
 	MaxAge:     3,
 	Level:      "warn",
 }
+
+// SetLogLevel set log level
+func SetLogLevel(level string) {
+	defaultConfig.Level = level
+}
+
+// SetLogPath set log path
+func SetLogPath(path string) {
+	defaultConfig.LogPath = path
+}
+
+// GetLogConfig get log config
+func GetLogConfig() *OutputConfig {
```

Comment:
It is not appropriate to directly expose objects for external settings. It is best to add methods to operate, and then add validity checks to the set values in the methods